docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -90,6 +90,61 @@ Once the initial enrollment has completed, and if Teleport Identity
 is enabled, the enrollment workflow will guide you through enabling SCIM
 provisioning as well.
 
+## Configure Access List Synchronization
+
+If Teleport Identity is enabled, the enrollment workflow will also guide you to configure syncing of
+user groups and direct application assignments as Access Lists. This will ensure
```

Comment:
I still believe that access list is akin to "role" or "resource" and is not a product name or proper noun.

@ptgott can you advise?

Reply:
As of now, the docs are sort of on the fence:

```
$ grep -hRio "access request" docs/pages | sort | uniq -c
 479 Access Request
   4 Access request
  26 access request
$ grep -hRio "access list" docs/pages | sort | uniq -c
  92 Access List
   1 Access list
  46 access list
```

I'm also a little torn. On the one hand, these are unique features of Teleport that we want to promote. On the other hand, these also make sense as general English terms, and capitalizing too many terms can be distracting.

I'll open an issue so we can settle this separately!

Reply:
Works for me!